docs/topics/api/addons.rst

```diff
@@ -44,6 +44,7 @@ This endpoint allows you to search through public add-ons.
     :query string appversion: Filter by application version compatibility. Pass the full version as a string, e.g. ``46.0``. Only valid when the ``app`` parameter is also present.
     :query string author: Filter by exact author username. Multiple author names can be specified, separated by comma(s), in which case add-ons with at least one matching author are returned.
     :query string category: Filter by :ref:`category slug <category-list>`. ``app`` and ``type`` parameters need to be set, otherwise this parameter is ignored.
+    :query string exclude_addons: Exclude add-ons by ``slug`` or ``id``. Multiple add-ons can be specified, separated by comma(s).
```

Comment:
Stupid question just by reading the docs: Can `slug` and `id` be mixed? 😀 (which I know is explicitly tested in the unittest below)

Reply:
Do you think we need to word that differently to make it clear it's possible? IMHO it's not really worth it, it's supported but we don't really want to encourage it.

Reply:
hmm, we definitely don't want to encourage that. Maybe ignore me and let it the way it is.